src/main/resources/alfresco/minimal-context.xml

```diff
@@ -1,48 +1,50 @@
-    <!--
-         Import the core parts of the context. They are required
-         for anything to work
-    -->
-    <import resource="classpath:alfresco/application-context-core.xml" />
-
-    <import resource="classpath*:alfresco/mt/mt-*context.xml"/>
-        
-    <!-- required to bring up the search subsystem -->
-    <import resource="classpath:alfresco/opencmis-context.xml" />
-    <import resource="classpath:alfresco/thumbnail-service-context.xml"/>
-    <import resource="classpath:alfresco/rendition-services-context.xml"/>
-    <import resource="classpath:alfresco/rendition-services2-context.xml"/>
-
-    <import resource="classpath*:alfresco/messaging-context.xml" />
-
-    <!-- Required by RepositoryStartStopTest which may apply the CalendarAllDayEventDatesCorrectingPatch -->
-    <import resource="classpath:alfresco/calendar-services-context.xml"/>
-
-    <!--
-         Import all modules and related components.
-    -->
-    <import resource="classpath*:alfresco/module-context.xml" />
-	
-    <!--
-         Import of the minimal extensions and bean overrides.
-    -->
-    <import resource="classpath*:alfresco/extension/dev-context.xml" />
-    <import resource="classpath*:alfresco/extension/test-context.xml" />
-    
-    <!-- Dummy bootstrap to stop minimal-context tests failing -->
-    <!--
-         TODO: minimal-context tests should not have the enterprise config
-         resources on the classpath.
-     -->
-    <bean id="clusteringBootstrap" class="java.lang.String"/>
-    
-    <bean id="ClusterService" class="java.lang.String"/>
-</beans>
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- This is the smallest XML configuration that will manage to start -->
+<!-- Many parts of the application are missing though, so don't expect -->
+<!--  to be able to do much more than run core unit tests with it -->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <!--
+         Import the core parts of the context. They are required
+         for anything to work
+    -->
+    <import resource="classpath:alfresco/application-context-core.xml" />
+
+    <import resource="classpath*:alfresco/mt/mt-*context.xml"/>
+        
+    <!-- required to bring up the search subsystem -->
+    <import resource="classpath:alfresco/opencmis-context.xml" />
+    <import resource="classpath:alfresco/thumbnail-service-context.xml"/>
+    <import resource="classpath:alfresco/rendition-services-context.xml"/>
+    <import resource="classpath:alfresco/rendition-services2-context.xml"/>
+
+    <import resource="classpath*:alfresco/messaging-context.xml" />
+
+    <!-- Required by RepositoryStartStopTest which may apply the CalendarAllDayEventDatesCorrectingPatch -->
+    <import resource="classpath:alfresco/calendar-services-context.xml"/>
```

Comment:
The only change to this file: I added lines 28 and 29. Apologies for IntelliJ changing every single line.